### PR TITLE
style(frontend): Change icons for the Hero back and menu buttons

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We update the icons for the Back button and the Context Menu in the Hero.

Screenshot 2024-10-16 at 13 17 10

### CODE REVIEW

**Review by peterpeterparker — COMMENTED** [review]
I would suggest to remove `[OISY-62]` in the title of the PR. Jira is closed source so adding those information that prominently feels a bit noisy in term of open source. WDYT?

**Comment by AntonioVentilii-DFINITY** [comment]
@peterpeterparker about the `[OISY-62]`, that is mostly because Jira auto-link the PRs to the ticket, so it is easier to follow-up in Jira itself. It is mostly for our (my) own internal management of tasks

File: src/frontend/src/lib/components/core/Back.svelte [hunk 1]
```diff
@@ -17,5 +19,12 @@
 
 <button
 	class={`text-${color} pointer-events-auto flex gap-0.5 font-bold`}
-	on:click={() => back({ pop: nonNullish(fromRoute) })}><IconBack /> {$i18n.core.text.back}</button
+	on:click={() => back({ pop: nonNullish(fromRoute) })}
 >
+	{#if onlyArrow}
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Back.svelte:25]: Right! Check last commit, i used `visually-hidden` , WDYT? Or shall i put a `aria-label`?

**Comment by peterpeterparker** [comment]
> It is mostly for our (my) own internal management of tasks

I meant prominently. If for example it's linked in the description, for sure I'm fine with it. I just meant that so expressively, in the PR title, feels really corporate and not open source.

**Comment by AntonioVentilii-DFINITY** [comment]
@peterpeterparker I didn't know it would work in the description too! Changed!

**Comment by AntonioVentilii-DFINITY** [comment]
[OISY-62]

File: src/frontend/src/lib/components/core/Back.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/core/Back.svelte:26]: Nice!

File: src/frontend/src/lib/components/icons/IconBackArrow.svelte [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!-- source: ISC Lucide - please visit https://lucide.dev/license -->
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/icons/IconBackArrow.svelte:1]: Unrelated: if we have a sub-folder `iconly`, should we have a `lucide` subfolder under icons?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/icons/IconBackArrow.svelte (hunk 2 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/icons/IconBackArrow.svelte:1]: yes, makes sense!

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/icons/IconBackArrow.svelte:1]: done:

- PR #2942

**Pull request merged**